src/typings.d.ts

```diff
@@ -151,6 +151,13 @@ interface ClipParams {
    *  - See:      https://tonejs.github.io/docs/13.8.25/Volume
    */
   volume?: number;
+  /**
+   * Boolean parameter to trigger offline rendering. Use this to help
+   * If true, `scribbletune.clip` returns a `Tone.Player` with a buffer containing a pre-rendered sound of the sequence
+   * If alse, it returns a `Tone.Sequence` which does live rendering.
```

Comment:
Minor typo `alse`

Reply:
Solved in https://github.com/scribbletune/scribbletune/pull/113/commits/1ff91750cb3dad5b79bff216af47fcc83b9f781c